lib/widgets/nutrition/forms.dart

```diff
@@ -137,7 +163,77 @@ class MealItemForm extends StatelessWidget {
         child: Column(
           children: [
             IngredientTypeahead(_ingredientIdController, _ingredientController),
+            ElevatedButton(
```

Comment:
is it possible to add this to ingredient typeahead? Ideally we would have a single widget where we could either type a name or scan a product. We can add a barcode scanner as a suffixIcon that opens the scanner (the search icon would go to preffixIcon). If that's not possible because the typeahead logic interferes with it, we can leave it as a separate button